client/src/pages/donate-other.js

```diff
@@ -0,0 +1,214 @@
+            </p>
+            <hr />
+            <h3>Donate through a payroll deduction</h3>
+            <p>
+              In the US and Canada, some employers have a convenient way to give
+              to freeCodeCamp through a payroll deduction. Ask your employer if
+              they can do this, and have them send any necessary paperwork to:
+            </p>
+            <p>
+              Free Code Camp, Inc.<br />7800 W Hefner Rd, PO BOX 721024<br />Oklahoma
+              City, Oklahoma 73172<br />United States
+            </p>
+            <hr />
+            <h3>Donate cryptocurrency to freeCodeCamp</h3>
+            <p>
+              Below are our wallets where we can receive cryptocurrency donations.
+            </p>
+            <h4>Make a one-time Bitcoin donation</h4>
+            <p className='negative-15'>
+              Our Bitcoin wallet is{' '}
+              <code>12skYi7aMCjDUdrVdoB3JjZ77ug8gxJfbL</code>
+            </p>
+            <h4>Make a one-time Ethereum donation</h4>
+            <p className='negative-15'>
+              Our Ethereum wallet is{' '}
+              <code>0x0ADbEf2471416BD8732cf0f3944294eE393CcAF5</code>
+            </p>
+            <h4>Make a one-time Litecoin donation</h4>
+            <p className='negative-15'>
+              Our Litecoin wallet is{' '}
+              <code>LKu8UG8Z1nbTxnq9Do96PsC3FwbNtycf3X</code>
+            </p>
+            <h4>Make a one-time Bitcoin Cash donation</h4>
+            <p className='negative-15'>
+              Our Bitcoin Cash wallet is{' '}
+              <code>1EBxPEJWrGZWxe2UayyAsnd5VsRg5H9xfu</code>
+            </p>
+            <hr />
+            <h3>Donate to freeCodeCamp by mailing us a check</h3>
+            <p>Our mailing address is:</p>
+            <p>
+              Free Code Camp, Inc.<br />7800 W Hefner Rd, PO BOX 721024<br />Oklahoma
+              City, Oklahoma 73172<br />United States
+            </p>
+            <hr />
+            <h3>Donate Stock to freeCodeCamp</h3>
+            <p>
+              If you want to donate stocks to freeCodeCamp, please email us at{' '}
+              <a href='mailto:team@freecodecamp.org'>team@freecodecamp.org</a>.
+            </p>
+            <hr />
+            <h3>Legacy Gift</h3>
+            <p>
+              You can help future generations of learners by listing freeCodeCamp
+              in your will or living trust. If you're interested in doing this,
+              email Quincy directly and we can discuss this:{' '}
+              <a href='mailto:quincy@freecodecamp.org'>quincy@freecodecamp.org</a>.
+            </p>
+            <hr />
+            <h3>
+              Thank you for supporting our nonprofit and the freeCodeCamp
+              community.
+            </h3>
+            <Spacer />
+            <div className='text-center'>
+              <a href='/donate'>Or donate using a Credit or Debit Card.</a>
```

Comment:
Please use the `Link` component provided in the gatsby package, it saves us from a full page reload.